src/styles/blocks/page.scss

```diff
@@ -0,0 +1,55 @@
+.html {
+  font-family: arial, helvetica, sans-serif;
+}
+
+.body {
+  padding-top: 50px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $day-gap;
+  padding: $calendar-padding;
+  max-width: $calendar-max-width;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.